src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/DocumentSymbols/RemoteDocumentSymbolService.cs

```diff
@@ -23,6 +24,7 @@ protected override IRemoteDocumentSymbolService CreateService(in ServiceArgs arg
     }
 
     private readonly IDocumentSymbolService _documentSymbolService = args.ExportProvider.GetExportedValue<IDocumentSymbolService>();
+    private readonly IClientCapabilitiesService _remoteClientCapabilitiesService = args.ExportProvider.GetExportedValue<IClientCapabilitiesService>();
```

Comment:
Consider renaming this field to `_clientCapabilitiesService`.

Reply:
renamed